torchbenchmark/util/framework/detectron2/__init__.py

```diff
@@ -43,7 +43,10 @@ def install_model_weights(model_name, model_dir):
 def pip_install_requirements():
     requirements_file = os.path.join(CURRENT_DIR, "requirements.txt")
     subprocess.check_call(
-        [sys.executable, "-m", "pip", "install", "-q", "-r", requirements_file]
+        [sys.executable, "-m", "pip", "install", "-q", "wheel", "numpy", "cython"] # Build-time requirements
```

Comment:
In order to be compatible with the upstream pytorch build-time numpy version, we have to pin the numpy version here: https://github.com/pytorch/benchmark/blob/main/requirements.txt#L20.

Can we assume that numpy has been installed in the environment and remove numpy here?

Reply:
Now it's removed.